src/com/company/Parse.java

```diff
@@ -0,0 +1,120 @@
+package com.company;
+
+import org.eclipse.cdt.core.dom.ast.ExpansionOverlapsBoundaryException;
+import org.eclipse.cdt.core.dom.ast.IASTNode;
+import org.eclipse.cdt.core.dom.ast.IASTTranslationUnit;
+import org.eclipse.cdt.core.dom.ast.gnu.cpp.GPPLanguage;
+import org.eclipse.cdt.core.index.IIndex;
+import org.eclipse.cdt.core.parser.*;
+import org.eclipse.cdt.internal.core.dom.parser.cpp.*;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Parse {
+
+    private IASTTranslationUnit translationUnit; //current translation unit node
+    public HashMap<String, Contents> repo;
```

Comment:
Will this work?
```suggestion
    public HashMap<String, Contents> repo = new HashMap<>();
```
(and delete the initialization in the constructor)

If the value doesn't depend on any constructor argument, we might as well initialize it on declaration.